File: src/main/java/mods/eln/integration/minetweaker/MinetweakerIntegration.java
package mods.eln.integration.minetweaker;

import minetweaker.MineTweakerAPI;
import mods.eln.integration.minetweaker.machines.Compressor;
import mods.eln.integration.minetweaker.machines.Macerator;
import mods.eln.integration.minetweaker.machines.Magnetizer;
import mods.eln.integration.minetweaker.machines.PlateMachine;

public class MinetweakerIntegration {
	public static MinetweakerIntegration instance = new MinetweakerIntegration();

	
	public void initialize(){
		MineTweakerAPI.registerClass(Macerator.class);
		MineTweakerAPI.registerClass(Compressor.class);
		MineTweakerAPI.registerClass(Magnetizer.class);
		MineTweakerAPI.registerClass(PlateMachine.class);
	}
	
}

File: src/main/java/mods/eln/integration/minetweaker/MinetweakerIntegration.kt
package mods.eln.integration.minetweaker

import minetweaker.MineTweakerAPI
import mods.eln.integration.minetweaker.machines.Compressor
import mods.eln.integration.minetweaker.machines.Macerator
import mods.eln.integration.minetweaker.machines.Magnetizer
import mods.eln.integration.minetweaker.machines.PlateMachine

object MinetweakerIntegration {

    fun initialize() {
        MineTweakerAPI.registerClass(Macerator::class.java)
        MineTweakerAPI.registerClass(Compressor::class.java)
        MineTweakerAPI.registerClass(Magnetizer::class.java)
        MineTweakerAPI.registerClass(PlateMachine::class.java)
    }

}
